examples/src/Viewer.tsx

```diff
@@ -1035,6 +1035,7 @@ class Viewer extends React.Component<InputParams, ViewerState> {
                         onError={this.onError}
                         backgroundColor={[0, 0, 0]}
                         lockedCamera={false}
+                        disableCache={false}
```

Comment:
Nit: Slightly prefer inverse of this, like `enableCache` or `cacheEnabled`? It's a little easier to reason about affirmative names!

Reply:
disable > enable was by @meganrm request from meeting yesterday!

Reply:
Oh, interesting... @meganrm I've generally seen affirmative names be convention for booleans, could you give some context for why you wanted `disableCache` here? :0

Reply:
because it's an optional prop you'd only use to disable the cache. The default behavior is to have a cache and therefore not have the prop. I get what you're saying about readability, but this props is basically going to not be there at all or be `disableCache={true}` (or you can do shorthand to just have `disableCache` which is clearer to me that always have `enableCache={false}` as the only way the prop is really used

Reply:
also, how this prop is then handled in the internals can be different: ie, whatever is more readable there. To me this is similar to like `disabled` as a prop on a button.